docs/install/knative-with-operators.md

```diff
@@ -8,7 +8,9 @@ Knative provides operators as tools to install, configure and manage Knative. Th
 uninstall Knative using Knative operators.
 
 Each component in Knative has a separate operator for installation and configuration. This means that there is a [Serving operator](https://github.com/knative/serving-operator)
-and an [Eventing operator](https://github.com/knative/eventing-operator), and you can choose to install one or both independently.
+and an [Eventing operator](https://github.com/knative/eventing-operator). Knative also releases both of them together as
+the [All-In-One operator](https://github.com/knative-sandbox/operator). You can choose to install them in one bundle or
+separately.
```

Comment:
I'd rewrite this to prefer the all-in-one bundle, and make it clear that it is `all in one` XOR `eventing + serving`.

```md
New in 0.14, Knative offers a [combined operator for managing both
Serving and Eventing](https://github.com/knative-sandbox/operator).
For release 0.14, you may **alternatively** choose to install the
[Serving operator](https://github.com/knative/serving-operator) and
[Eventing operator](https://github.com/knative/eventing-operator)
instead of the combined operator.
```